docs/register-intune-device-id.md

```diff
@@ -2,43 +2,42 @@
+2. Hover over your account name and click **My Account**.
 
-> **Note**
->- Ensure you have completed steps 1 and 2. Refer to [SEED onboarding](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/onboard-device/onboard-device-to-seed) for step-by-step instructions to onboard a device to SEED.
+![view-account](assets/images/onboarding/po-non-se/view-account-or-profile.png)
 
+3. Click **Onboard device to SEED**.
+
+![onboard-device-to-seed](assets/images/seed/licence-assigned-onboard-device.png)
 
-4. Enter the Intune device ID in this dialog.
+The **SEED - Onboard Device** dialog is displayed.
 
-> **Note**
->- Ensure there are no spaces at the beginning and at the end of the Intune device ID.
+4. **Enter the Intune Device ID** without any spaces .
 
-<kbd>![instructions-to-onboard-device-to-seed](assets/images/seed/onboard-device.png)</kbd>
+![instructions-to-onboard-device-to-seed](assets/images/seed/onboard-device.png)
 
 5. Click **Submit**.
 
 After you submit the Intune device ID, the following success message is displayed.
 
-<kbd>![onboard-device-success](assets/images/seed/onboard-device-success.png)</kbd>
-
- >**Note**<br>
- >- It might take up to 60 minutes for Intune to update your device with the required softwares and configurations. During this time, the device onboarding status is **Pending**.
- >- After the device is updated with all the required softwares and configurations, the device onboarding status changes to **Onboarded** and you will receive a successfully onboarded email.
-
-6. Refer to [SEED documentation](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/post-onboarding-instructions/post-onboarding-steps-and-verification) and complete the post onboarding instructions for SEED.
+![onboard-device-success](assets/images/seed/onboard-device-success.png)
 
-?> If you are a GCC 1.0 user, refer to the [post onboarding instructions for GCC 1.0 users](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/post-onboarding-instructions/gcc-1.0-users).
+>**Note**<br>- It might take up to 60 minutes for Intune to update your device with the required software and configurations. During this time, the device onboarding status is Pending.<br>- After the device is updated with all the required software and configurations, the device onboarding status changes to **Onboarded** and you will receive a successfully onboarded email.<br>- Refer to [SEED documentation](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/post-onboarding-instructions/post-onboarding-steps-and-verification) and complete the post onboarding instructions for SEED.<br>- If you are a GCC 1.0 user, refer to the [post onboarding instructions for GCC 1.0 users](https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices/#/post-onboarding-instructions/gcc-1.0-users).
```

Comment:
I'm sure you already know that there's some changes to the SEED onboarding (i.e. step functions). I'll let you decide if this section need to be updated.

Reply:
Yes i am aware :-),  i will take that as a separate change, I will need some time to update the doc for the new changes w.r.t SEED onboarding status.